templates/CSI/cephfs/storageclass.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: storage.k8s.io/v1
```

Comment:
same

Reply:
@ebenahar the contents and fields of storageclass.yaml  are different for rbd and cephfs. So wouldn't it be better to have distinct files for rbd and cephfs ?

Reply:
@nehaberry +1